File: CSES/Python/Introductory/repetitions.py
dna = input()

longest = 1
curr = 1
prev = dna[0]
for i in range(1,len(dna)):
    if dna[i] != dna[i-1]:
        curr = 0
    curr += 1
    longest = max(longest,curr)

print(longest)

File: CSES/C++/Introductory/repetitions.cpp
#include <bits/stdc++.h>
using namespace std;
using ll = long long;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int longest = 1;
    int curr = 1;
    char last; cin >> last;
    char c;
    while (cin >> c)
    {
        if (c == last) curr++;
        else curr = 1;
        last = c;
        longest = max(longest, curr);
    }
    cout << longest << '\n';
}